pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/diagnostic.go

```diff
@@ -33,7 +33,6 @@ import (
 // https://github.com/openshift/origin-aggregated-logging
 type AggregatedLogging struct {
 	masterConfig      *configapi.MasterConfig
-	loggingURL        string
 	MasterConfigFile  string
```

Comment:
Now that we do not depend on masterConfig.AssetConfig.LoggingPublicURL, you can remove MasterConfigFile?